### PR TITLE
p1-grid

### PR DESCRIPTION
**Pull request by sumitk-singh**
## codes to review for-
- two items in grid
- three items in grid

### CODE REVIEW

**Review by itsmebipul00 — APPROVED** [review]
Suggestion : You could do these separately with grid-template-columns/rows as well. Grid template areas defines both columns and rows. Good to merge

File: components/Grid.css [hunk 1]
```diff
@@ -36,4 +40,64 @@
     position: absolute;
     margin-left: 1.563rem;
     margin-top: 5px;
-}
\ No newline at end of file
+}
+
+/* start of grid-two-items */
```
Comment (Megha-Joshi) [inline components/Grid.css:45]: Nice use of comments.

File: components/Grid.css [hunk 2]
```diff
@@ -36,4 +40,64 @@
...
+    display: grid;
+    text-align: center;
+    gap: 1rem;
+    grid-template-areas:
+        "grid-two-items-item-1 grid-two-items-item-2";
+    width: 100%;
+}
+
+.grid-two-items-item-1 {
+    grid-area: grid-two-items-item-1;
```
Comment (Megha-Joshi) [inline components/Grid.css:56]: Suggestion: You can also use grid-template-row and column.

**Review by Megha-Joshi — APPROVED** [review]
- Nice use of comments.
- Relevant class names.
- Very clean code.
- Good to merge

**Pull request merged**